tests/test_git_repo_scanner.py

```diff
@@ -98,18 +98,14 @@ def test_extra_exclusions_get_added(self, mock_load: mock.MagicMock):
     def test_extra_signatures_get_added(self, mock_load: mock.MagicMock):
         mock_load.return_value = (
             self.data_dir / "pyproject.toml",
-            {
-                "exclude_signatures": ["foo", "bar"],
-                "exclude_findings": [{"signature": "foo/bar", "reason": "test reason"}],
-            },
+            {"exclude_signatures": ["foo", "bar"]},
         )
+        self.global_options.exclude_signatures = ("foo", "bar")
         test_scanner = scanner.GitRepoScanner(
             self.global_options, self.git_options, str(self.data_dir)
         )
         test_scanner.load_repo("../tartufo")
-        self.assertEqual(
-            sorted(test_scanner.excluded_findings), ["bar", "foo", "foo/bar"]
-        )
+        self.assertEqual(sorted(test_scanner.excluded_signatures), ["bar", "foo"])
```

Comment:
```suggestion
        self.assertCountEqual(test_scanner.excluded_signatures, ["bar", "foo"])
```
Handy trick for comparing lists in unit tests: This makes sure that all of the elements match, but doesn't care about the order of those elements.